src/components/pages/Registration/InstructorFlow_Step2.js

```diff
@@ -0,0 +1,162 @@
+      .validate(value)
+      .then(() => setFormErrors({ ...formErrors, [name]: '' }))
+      .catch(err => setFormErrors({ ...formErrors, [name]: err.errors[0] }));
+  };
+
+  const inputChange = (name, value) => {
+    validate(name, value);
+    setFormValues({
+      ...formValues,
+      [name]: value,
+    });
+  };
+
+  const formSave = () => {
+    // TODO
+  };
+
+  const onSubmit = evt => {
+    evt.preventDefault();
+    formSave();
+    setFormValues(initialValues);
+  };
+
+  const onChange = evt => {
+    const { name, value } = evt.target;
+    inputChange(name, value);
+  };
+
+  useEffect(() => {
+    InstructorFormSchema.isValid(formValues).then(valid => setDisabled(!valid));
+  }, [formValues]);
+
+  return (
+    <div>
+      <div className="reg-content-container">
+        <div className="reg-form-header">
+          <h1>Instructor Account Info</h1>
+          <p className="indicator">*indicates required field</p>
+        </div>
+
+        <form className="basic-form" onSubmit={onSubmit}>
+          <label>
+            <input
+              name="name"
+              type="text"
+              placeholder="*Name"
+              value={formValues.name}
+              onChange={onChange}
+            />
+          </label>
+
+          <label>
+            <input
+              name="email"
+              type="email"
+              placeholder="*Email"
+              value={formValues.email}
+              onChange={onChange}
+            />
+          </label>
+
+          <label>
+            <input
+              name="location"
+              type="text"
+              placeholder="*City,State"
+              value={formValues.location}
+              onChange={onChange}
+            />
+          </label>
+
+          <label>
+            <input
+              name="phone"
+              type="tel"
+              placeholder="*Phone Number"
+              value={formValues.phone}
+              onChange={onChange}
+            />
+          </label>
+
+          <label>
+            <input
+              name="education"
+              type="text"
+              placeholder="*Education (include degree and University)"
+              value={formValues.education}
+              onChange={onChange}
+            />
+          </label>
+
+          <label>
+            <input
+              name="tech"
+              type="text"
+              placeholder="Technical Experience"
+              value={formValues.tech}
+              onChange={onChange}
+            />
+          </label>
+
+          <label>
+            <input
+              name="notes"
+              type="text"
+              placeholder="Notes (Use this space to provide any additional context or relevant experience)"
+              value={formValues.notes}
+              onChange={onChange}
+            />
+          </label>
+
+          <div className="SaveButton">
+            <button disabled={disabled}>Save</button>
```

Comment:
I wonder if we should change this to "Save & Submit" or something like that to make it clear they are finalizing the registration process?